Typescript error in Geolocation

**Describe the bug**
Typescript 4.8 doesn't like the types:

```
﻿node_modules/ol/Geolocation.d.ts:103:5 - error TS2416: Property 'on' in type 'Geolocation' is not assignable to the same property in base type 'BaseObject'.
  Type 'GeolocationOnSignature<EventsKey>' is not assignable to type 'ObjectOnSignature<EventsKey>'.
    Type 'GeolocationOnSignature<EventsKey>' is not assignable to type 'OnSignature<EventTypes, BaseEvent, EventsKey>'.
      Types of parameters 'type' and 'type' are incompatible.
        Type 'EventTypes' is not assignable to type '"change"'.
          Type '"error"' is not assignable to type '"change"'.

103     on: GeolocationOnSignature<import("./events").EventsKey>;
        ~~
node_modules/ol/Geolocation.d.ts:107:5 - error TS2416: Property 'once' in type 'Geolocation' is not assignable to the same property in base type 'BaseObject'.
  Type 'GeolocationOnSignature<EventsKey>' is not assignable to type 'ObjectOnSignature<EventsKey>'.

107     once: GeolocationOnSignature<import("./events").EventsKey>;
        ~~~~
node_modules/ol/Geolocation.d.ts:111:5 - error TS2416: Property 'un' in type 'Geolocation' is not assignable to the same property in base type 'BaseObject'.
  Type 'GeolocationOnSignature<void>' is not assignable to type 'ObjectOnSignature<void>'.
    Type 'GeolocationOnSignature<void>' is not assignable to type 'OnSignature<EventTypes, BaseEvent, void>'.
      Types of parameters 'type' and 'type' are incompatible.
        Type 'EventTypes' is not assignable to type '"change"'.
          Type '"error"' is not assignable to type '"change"'.

111     un: GeolocationOnSignature<void>;
        ~~
```

**To Reproduce**
Include any ol stuff in a Typescript project with `skipLibCheck: false`, the above type errors are reported. With `skipLibCheck: true` it works because it's then not checked.

**Expected behavior**
Compile with `skipLibCheck: false` should pass

--- a/src/ol/Geolocation.js
+++ b/src/ol/Geolocation.js
@@ -84,10 +84,10 @@ export class GeolocationError extends BaseEvent {
 
 /***
  * @template Return
- * @typedef {import("./Observable").OnSignature<'change', import("./events/Event.js").default, Return> &
- *   import("./Observable").OnSignature<GeolocationObjectEventTypes, import("./Object").ObjectEvent, Return> &
+ * @typedef {import("./Observable").OnSignature<GeolocationObjectEventTypes, import("./Object").ObjectEvent, Return> &
  *   import("./Observable").OnSignature<'error', GeolocationError, Return> &
- *   import("./Observable").CombinedOnSignature<import("./Observable").EventTypes|GeolocationObjectEventTypes, Return>} GeolocationOnSignature
+ *   import("./Observable").CombinedOnSignature<import("./Observable").EventTypes|GeolocationObjectEventTypes, Return> &
+ *   import("./Observable").OnSignature<import("./Observable").EventTypes, import("./events/Event.js").default, Return>} GeolocationOnSignature
  */
 
 /**

